proto/sanitizer.proto

```
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package sanitizer;
```

Buf (1.55.1)

Files with package "sanitizer" must be within a directory "sanitizer" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)